### PR TITLE
Fix JSDocs

### PR DESCRIPTION
**Pull request by vlad-zhukov**

### CODE REVIEW

**Comment by icambron** [comment]
The `@type` ones are right. See [https://esdoc.org/manual/tags.html#-type](here). Upper case object appears to be right too; see [here](https://esdoc.org/manual/tags.html#-return).

File: src/settings.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ let now = () => new Date().valueOf(),
 export class Settings {
   /**
    * Get the callback for returning the current timestamp.
-   * @type {function}
+   * @return {function}
```
Comment (icambron) [inline src/settings.js]: I think you missed reverting these

File: src/duration.js [hunk 2]
```diff
@@ -568,55 +568,55 @@ export class Duration {
 
   /**
    * Get the years.
-   * @return {number}
+   * @type {number}
```
Comment (icambron) [inline src/duration.js]: 👍

**Comment by vlad-zhukov** [comment]
Thanks you very much for helping! 🍕 I was loaded at work and forgot about this. 😞

**Pull request merged**